addons/fleet/models/fleet_fleet.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, fields, models
+
+class FleetFleet(models.Model):
+    _name = 'fleet.fleet'
```

Comment:
The difference between ``fleet.fleet`` and ``fleet.vehicle`` is not clear. 

Wouldn't it be clearer to name the model ``fleet.category`` or something like this?